meterfs/test_meterfs_openclose.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs opening and closing tests group
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Tomasz Korniluk
+ *
+ *
+ * %LICENSE%
+ */
+
+#include "test_meterfs_common.h"
+
+TEST_GROUP(meterfs_openclose);
+
+
+TEST_SETUP(meterfs_openclose)
+{
+	test_common.ids[0] = 0;
+	memset(test_common.buff, 0, sizeof(test_common.buff));
+}
+
+
+TEST_TEAR_DOWN(meterfs_openclose)
+{
+	TEST_ASSERT_EQUAL(0, test_fileEraseAll());
+}
+
+
+/* test case of opening and closing non existing files */
+TEST(meterfs_openclose, no_files)
+{
+	int i;
+
+	for (i = 0; i < 255; ++i) {
+		sprintf(test_common.buff, "/file%d", i);
+		TEST_ASSERT_LESS_THAN_MESSAGE(0, test_fileOpen(test_common.buff), test_common.buff);
```

Comment:
I wodner if we shouldn't check if valid error is returned here, not any error in place like this. What do you think?

Reply:
Done. I implemented this kind of treatment in rest of the code.